inc/class-cachify.php

```diff
@@ -371,21 +371,21 @@ public static function on_uninstall() {
 	}
 
 	/**
-	 * Uninstalling of the plugin for MU and network.
+	 * Uninstalling of the plugin for WPMS site.
 	 *
 	 * @since   1.0
 	 * @change  1.0
 	 *
-	 * @param integer $id  Blog ID.
+	 * @param WP_Site $old_site Old Site object.
 	 */
-	public static function uninstall_later( $id ) {
+	public static function uninstall_later( $old_site ) {
```

Comment:
The change for `install_later` add the type hint ` WP_Site $new_site`. Was this omitted in this place intentionally?

However, this is strictly speaking public API. So I'd suggest to make it backwards compatible for now, i.e. drop the type hint and use implicit union type `int|WP_Site`. We only need the ID anyway, so we can simply extract it, if we get a `WP_Site`  instance and passthrough integers.